pkg/restore/restore.go

```diff
@@ -264,14 +280,16 @@ func (kr *kubernetesRestorer) Restore(
 		restoredItems:              make(map[velero.ResourceIdentifier]struct{}),
 		renamedPVs:                 make(map[string]string),
 		pvRenamer:                  kr.pvRenamer,
-		discoveryHelper:            kr.discoveryHelper,
+		discoveryHelper:            discoveryHelper,
 		resourcePriorities:         kr.resourcePriorities,
-		resourceRestoreHooks:       resourceRestoreHooks,
-		hooksErrs:                  make(chan error),
-		waitExecHookHandler:        waitExecHookHandler,
-		hooksContext:               hooksCtx,
-		hooksCancelFunc:            hooksCancelFunc,
-		restoreClient:              kr.restoreClient,
+		// Because we don't use hooks.
```

Comment:
I assume they are not used, so it doesn't hurt to just set them anyway?

Reply:
I passed them in but this required comments in other sections so that they are not used. Hope that's fine :wink: